src/main/resources/fleet_mgmt_location.avro

```diff
@@ -0,0 +1,56 @@
+{
+    "namespace": "fleet_mgmt",
+    "name": "fleet_mgmt_location",
+    "type": "record",
+    "fields": [
+        {
+            "name": "vehicle_id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "range": {
+                        "min": 1000,
+                        "max": 9999
+                    }
+                }
+            }
+        },
+        {
+            "name": "latitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -50,
+                        "max": 70
+                    }
+                }
+            }
+        },
+        {
+            "name": "longitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -175,
+                        "max": 175
+                    }
+                }
+            }
+        },
+        {
+            "name": "timestamp",
+            "type": {
+                "type": "long",
+                "logicalType": "timestamp-millis",
+                "arg.properties": {
+                    "range": {
+                        "min": 1609459200000,
+                        "max": 1640995199000
+                    }
+                }
+            }
+        }
```

Comment:
optional: I kind of like the pattern used below in `gaming_games.created_date` since it's unbounded and steps every few seconds.  wdyt about changing to:
```
        {
            "name": "timestamp",
            "type": {
                "type": "long",
                "logicalType": "timestamp-millis",
                "arg.properties": {
                    "iteration": {
                        "start": 1609459200000,
                        "step": 100000
                    }
                }
            }
        }
```